### PR TITLE
prerelease notes for 0.19-alpha

### PR DESCRIPTION
**Pull request by mlubin**
After merging this I'll tag `v0.19-alpha`. I'm not certain that `add JuMP#v0.19-alpha` will work though.

@blegat @odow @joaquimg @joehuchette @juan-pablo-vielma @IssamT 

Please take a look and check that you agree with the lists of breaking changes/new features/known issues.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,102 @@
...
+- Starting values are separate from result values. Use `result_value` to query
+  the value of a variable in a solution. Use `start_value` and `set_start_value`
+  to get and set an initial starting point provided to the solver.
+
+- The data structures for affine and quadratic expressions `AffExpr` and
+  `QuadExpr` have changed. Internally, terms are stored in dictionaries instead
+  of lists. Duplicate coefficients can no longer exist. Accessors and iteration
+  methods have changed.
+
+- `JuMPNLPEvaluator` no longer includes the the linear and quadratic and
```
Comment (joehuchette) [inline NEWS.md]: duplicated "and quadratic"

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,102 @@
...
+- JuMP solver-independent callbacks have been replaced by solver-specific
+  callbacks. See your favorite solver for more details. (TODO: No
+  solver-specific callbacks are implemented yet.)
+
+- The `norm()` syntax is no longer recognized inside macros. Use the
+  `SecondOrderCone()` set instead. (TODO: This syntax is undocumented.)
+
+New features:
+
+- Support for deleting constraints and variables
```
Comment (joehuchette) [inline NEWS.md]: Nit: End with period (here and below).

File: NEWS.md (hunk 1 above)
Comment (blegat) [reply NEWS.md]: Also duplicated "the"

**Comment by blegat** [comment]
We could also mention that atm there is no automatic bridging between soc and quadratic while there was the conic <-> lpqp bridges with mpb but there are open prs for that

We could also advertise that JuMP extensibility has been improved as MOI is now extensible by design and JuMP macros are now extensible and a doc is coming

**Comment by mlubin** [comment]
> We could also mention that atm there is no automatic bridging between soc and quadratic while there was the conic <-> lpqp bridges with mpb but there are open prs for that

Is this a known issue or a breaking change (that it doesn't work anymore)?

**Comment by blegat** [comment]
> Is this a known issue or a breaking change (that it doesn't work anymore)?

Not sure which yet. For quadratic to SOC, https://github.com/JuliaOpt/MathOptInterface.jl/pull/483 does the non-homogeneous tranformation but I am not sure that the homogeneous one (i.e. `x^2 + y^2 <= z^2`) will be done for JuMP v0.19. For the SOC to quadratic, we are not sure to do it anymore because it would also work for NLP solvers: https://github.com/JuliaOpt/MathOptInterface.jl/pull/478#discussion_r211378894

**Pull request merged**